src/pages/RepoPage/RepoPage.tsx

```diff
@@ -310,6 +316,7 @@ function RepoPage() {
           isRepoPrivate={isRepoPrivate}
           isCurrentUserActivated={isCurrentUserActivated}
           testAnalyticsEnabled={repoOverview?.testAnalyticsEnabled}
+          isCurrentUserPartOfOrg={repoData?.isCurrentUserPartOfOrg || false}
```

Comment:
do we need the || false check here? if repoData?.isCurrentUserPartOfOrg is undefined it'll still be a falsey value

Reply:
yes if we want to explicitly set it to boolean, it's the pattern used in this component (see isRepoActivated, isRepoActive)

Reply:
Ah okay makes sense! Only thing I'd say then if we're looking for a boolean is we can utilize the !! syntax to cast a variable to its boolean equivalent

But that's a nit if anything

Reply:
sure thing!